src/agents/tools/twitterTools.ts

```diff
@@ -130,6 +130,16 @@ export const createFetchMyRecentTweetsAndRepliesTool = (twitterApi: TwitterApi)
     },
   });
 
+export const createLikeTweetTool = (twitterApi: TwitterApi) =>
+  new DynamicStructuredTool({
+    name: 'like_tweet',
+    description: 'Like a tweet that you find interesting but not worth responding to',
```

Comment:
```suggestion
    description: 'Like a tweet that you find interesting and in aligned with your conviction, regardless if you respond to it or not',
```
I don't believe it should be like OR reply, I usually like and reply, or like and not reply, or reply while not liking the tweet

Reply:
Great point!